tests/test_026_grid.py

```diff
@@ -134,6 +134,48 @@ def test_fillnan_da():
     assert not top[mask].isnull().any()
 
 
+def test_interpolate_gdf_to_array():
+    bgt = get_bgt()
+    bgt.geometry = bgt.centroid
+    bgt["values"] = range(len(bgt))
+
+    regis = get_regis()
+    ds = nlmod.to_model_ds(regis, model_ws=model_ws)
+    sim = nlmod.sim.sim(ds)
+    gwf = nlmod.gwf.gwf(ds, sim)
+    nlmod.gwf.dis(ds, gwf)
+
+    nlmod.grid.interpolate_gdf_to_array(bgt, gwf, field="values", method="linear")
```

Comment:
we should try to bring some more structure to the interpolation methods? There are quite a few, and I've been working on a new one, but finding them isn't as easy as I'd hoped.

Reply:
This notebook gives an overview of what is available: https://nlmod.readthedocs.io/en/stable/examples/07_resampling.html

Reply:
They are in https://nlmod.readthedocs.io/en/stable/examples/06_gridding_vector_data.html

I split the original notebook in a gridding_vector_data-notebook and a resample-notebook a long time ago.

Reply:
Thanks for the reminders, I saw the resampling notebook but I wasn't really looking for raster ops, but somehow missed the gridding_vector data notebook. I think the new work should probably get it's own notebook, but it is essentially the opposite direction of the gridding vectors notebook (but only for points). I'll commit it now so you guys can take a look.